cranelift/filetests/filetests/isa/x64/struct-ret.clif

```diff
@@ -96,8 +97,9 @@ block0(v0: i64):
 ;   movq %r15, (%rsp)
 ; block1: ; offset 0xc
 ;   movq %rdi, %r15
-;   movabsq $0, %rdx ; reloc_external Abs8 %f4 0
-;   callq *%rdx
+;   movabsq $0, %rax ; reloc_external Abs8 %f4 0
+;   movq %r15, %rdi
```

Comment:
This is a great example of something the redundant move eliminator would have fixed before.